packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -569,6 +557,7 @@ const DatePickerInput = (props) => {
     <Input
       {...additionalInputProps}
       refCallback={setDayRef}
+      ref={dateFormatOrder === DateUtil.dateOrder.DMY ? firstInputRefCallback : undefined}
```

Comment:
It would be helpful to add a comment explaining why passing both `ref` and `refCallback` for the time being and the issue we were seeing.

Reply:
I have added a comment explaining the reason here https://github.com/cerner/terra-framework/pull/1208/commits/286d1d030d422368c293acb30bd0bf2d3dfade5f